### PR TITLE
Fixes filtering on several columns

### PR DESCRIPTION
**Pull request by kevgliss**
Enables/Fixes filtering on:

- Tag Table -> Tag Type
- Task Table -> Creator, Owner, Due By, Incident Name, Incident Priority, Incident Type

### CODE REVIEW

File: src/dispatch/static/dispatch/src/task/Table.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
               :sort-by.sync="sortBy"
               :sort-desc.sync="descending"
             >
-              <template v-slot:item.creator="{ item }">
+              <template v-slot:item.creator.individual_contact.name="{ item }">
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/task/Table.vue:33]: Shouldn't this be `individual` not `individual_contact` based on the Participant model?

File: src/dispatch/static/dispatch/src/task/Table.vue [hunk 2]
```diff
@@ -44,7 +44,7 @@
                   Unknown
                 </v-chip>
               </template>
-              <template v-slot:item.owner="{ item }">
+              <template v-slot:item.owner.individual_contact.name="{ item }">
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/task/Table.vue:47]: Idem here.

File: src/dispatch/static/dispatch/src/task/Table.vue [hunk 3]
```diff
@@ -58,6 +58,12 @@
                   Unknown
                 </v-chip>
               </template>
+              <template v-slot:item.incident_priority.name="{ item }">
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/task/Table.vue:61]: I think `incident` is missing after `item.`

File: src/dispatch/static/dispatch/src/task/Table.vue [hunk 4]
```diff
@@ -58,6 +58,12 @@
...
+                {{ item.incident.incident_priority.name }}
+              </template>
+              <template v-slot:item.incident_type.name="{ item }">
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/task/Table.vue:64]: Same here.

File: src/dispatch/static/dispatch/src/task/Table.vue [hunk 5]
```diff
@@ -136,17 +142,17 @@ export default {
...
-        { text: "Incident Priority", value: "incident.incident_priority.name", sortable: false },
-        { text: "Incident Type", value: "incident.incident_type.name", sortable: false },
+        { text: "Incident Name", value: "incident.name", sortable: true },
+        { text: "Incident Priority", value: "incident_priority.name", sortable: true },
+        { text: "Incident Type", value: "incident_type.name", sortable: true },
         { text: "Status", value: "status", sortable: true },
-        { text: "Creator", value: "creator", sortable: true },
-        { text: "Owner", value: "owner", sortable: true },
+        { text: "Creator", value: "creator.individual_contact.name", sortable: true },
+        { text: "Owner", value: "owner.individual_contact.name", sortable: true },
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/task/Table.vue:150]: `individual` not `individual_contact`?

Comment (kevgliss) [reply src/dispatch/static/dispatch/src/task/Table.vue:150]: No, the actual model is individual contact (which is then resolved and used for filtering). We have a disconnect in that the actual model is called `individual_contact` but we refer to it as `individual` in most places.

File: src/dispatch/static/dispatch/src/task/Table.vue (hunk 1 above)
Comment (kevgliss) [reply src/dispatch/static/dispatch/src/task/Table.vue:33]: No, the slot name has to match the value defined as the value in the header value. It has no reflection on the actual item data structure.

File: src/dispatch/static/dispatch/src/task/Table.vue (hunk 2 above)
Comment (kevgliss) [reply src/dispatch/static/dispatch/src/task/Table.vue:47]: Same as above

File: src/dispatch/static/dispatch/src/task/Table.vue (hunk 3 above)
Comment (kevgliss) [reply src/dispatch/static/dispatch/src/task/Table.vue:61]: Same as above.

File: src/dispatch/static/dispatch/src/task/Table.vue (hunk 4 above)
Comment (kevgliss) [reply src/dispatch/static/dispatch/src/task/Table.vue:64]: Same as above.

File: src/dispatch/static/dispatch/src/task/Table.vue (hunk 1 above)
Comment (mvilanova) [reply src/dispatch/static/dispatch/src/task/Table.vue:33]: Got it.

File: src/dispatch/static/dispatch/src/task/Table.vue (hunk 5 above)
Comment (mvilanova) [reply src/dispatch/static/dispatch/src/task/Table.vue:150]: Yeah, it's a little bit confusing.

**Review by mvilanova — APPROVED** [review]

**Pull request merged**